### ISSUE TITLE
Negative Target

### ISSUE THREAD

**Issue by frrrizzz** [butor/blackbird#429]
Hi,

I don‘t understand the following:
In the config I have specified the spread target as 0.008 and the variable below (exit target or so, as 0.005, both default values).

Now BB entered the market at around the specified 0.8% spread, but since then shows a target of -0.64%.

Now ad far as I understand the arbitrage principle, we enter the market and some specified spread (the larger the more money you can make but the less likely the event is to occur). And we leave the market, ad soon ad the spread has reduced to some specified smaller Niveau. A negative spread of course means even a better deal in terms if money, but on the other hand also is very unlikely to occur in a market, where the spread was the other way round when we entered the market. 

By increasing the spread target, the exit target can be made zero or positive again, as the negative target probably is due to the fees of the exchanges, but shouldn‘t BB by default ensure, that no negative target can be created which can be very tough or even impossible to reach?

Kind regards,
Marcus

**Comment by monossido** [comment]
I don't agree; negative spread it's possible, there is no garantee about the spread trend.

**Comment by butor** [comment]
The basic principle of our arbitraging strategy is capturing _discrepancies_ between exchanges. In an efficient market, only one BTC/USD price should be offered across all the exchanges.

So when a divergence appears **and there is no outside reason for it** (e.g. exchange going bankrupt) then it is likely, but not certain, that the prices converge at some point in the near future.

That being said, bitcoin is still volatile and what we've seen is that sometimes prices swing between each other:



Time | Exh. A | Exch. B | A/B Spread
--: | --: | --: | --:
6am | 10,300 | 10,000 | Positive
9am | 10,500 | 10,000 | Positive
12pm | 9,900 | 10,000 | Negative
3pm | 9,900 | 10,200 | Negative
6pm | 9,900 | 9,800 | Positive

If users want to capture that kind of behavior, they can set their strategy parameters to capture a small spread and hoping that the spread will soon go the other way around. Negative spread is **only** useful when prices swing between each other in a volatile environment.
